.pipelines/e2e-tests-template.yaml

```diff
@@ -48,11 +48,34 @@ jobs:
           parameters:
             identitySubscriptionID: $(SUBSCRIPTION_ID)
 
+        - script: |
+            export REGISTRY="${REGISTRY:-$(REGISTRY_NAME).azurecr.io/k8s/csi/secrets-store}"
+
+            #This gets latest released chart(tgz) version from charts directory
+            export HELM_CHART_DIR=charts/$(ls -h charts | sort --version-sort --field-separator=- --key=6 | tail -n 1)
```

Comment:
This command should be 
```
export HELM_CHART_DIR=charts/$(ls -h charts/*.tgz | sort --version-sort --field-separator=- --key=6 | tail -n 1)
```

otherwise the result will be `index.yaml`

Reply:
The test in PR pipeline run appears to be working. But I'll update it for more accuracy.